teamcity: failed tests on master: test/TestLogic, test/TestLogic/default, test/TestLogic/default/crdb_internal, test/TestLogic/distsql, test/TestLogic/distsql/crdb_internal, testrace/TestLogic, testrace/TestLogic/default, testrace/TestLogic/default/crdb_internal, testrace/TestLogic/distsql, testrace/TestLogic/distsql/crdb_internal

The following tests appear to have failed:

[#332319](https://teamcity.cockroachdb.com/viewLog.html?buildId=332319):

```
--- FAIL: test/TestLogic (59.080s)
test_log_scope.go:78: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic305915677
test_log_scope.go:61: use -show-logs to present logs inline
--- FAIL: test/TestLogic/default (0.190s)
null
--- FAIL: test/TestLogic/default/crdb_internal (0.480s)
logic_test.go:1771: 
	testdata/logic_test/crdb_internal:174: 
	expected:
	    node_id component field  value
	    1       DB        URL    postgresql://root@127.0.0.1:<port>?sslcert=test_certs%2Fclient.root.crt&sslkey=test_certs%2Fclient.root.key&sslmode=verify-full&sslrootcert=test_certs%2Fca.crt
	    1       DB        Scheme postgresql
	    1       DB        User   root
	    1       DB        Host   127.0.0.1
	    1       DB        Port   <port>
	    1       DB        Path   ��
	    1       UI        URL    https://127.0.0.1:<port>
	    1       UI        Scheme https
	    1       UI        User   ��
	    1       UI        Host   127.0.0.1
	    1       UI        Port   <port>
	    1       UI        Path   ��
	    
	but found (query options: "colnames") :
	    node_id  component  field   value                                                                                                                                                                                       
	    1        DB         URL     postgresql://root@127.0.0.1:<port>?application_name=cockroach&sslcert=test_certs%2Fclient.root.crt&sslkey=test_certs%2Fclient.root.key&sslmode=verify-full&sslrootcert=test_certs%2Fca.crt  
	    1        DB         Scheme  postgresql                                                                                                                                                                                  
	    1        DB         User    root                                                                                                                                                                                        
	    1        DB         Host    127.0.0.1                                                                                                                                                                                   
	    1        DB         Port    <port>                                                                                                                                                                                      
	    1        DB         Path    ��                                                                                                                                                                                           
	    1        UI         URL     https://127.0.0.1:<port>                                                                                                                                                                    
	    1        UI         Scheme  https                                                                                                                                                                                       
	    1        UI         User    ��                                                                                                                                                                                           
	    1        UI         Host    127.0.0.1                                                                                                                                                                                   
	    1        UI         Port    <port>                                                                                                                                                                                      
	    1        UI         Path    ��                                                                                                                                                                                           
	
logic_test.go:1804: testdata/logic_test/crdb_internal:191: too many errors encountered, skipping the rest of the input
--- FAIL: test/TestLogic/distsql (0.440s)
null
--- FAIL: test/TestLogic/distsql/crdb_internal (0.900s)
logic_test.go:1771: 
	testdata/logic_test/crdb_internal:174: 
	expected:
	    node_id component field  value
	    1       DB        URL    postgresql://root@127.0.0.1:<port>?sslcert=test_certs%2Fclient.root.crt&sslkey=test_certs%2Fclient.root.key&sslmode=verify-full&sslrootcert=test_certs%2Fca.crt
	    1       DB        Scheme postgresql
	    1       DB        User   root
	    1       DB        Host   127.0.0.1
	    1       DB        Port   <port>
	    1       DB        Path   ��
	    1       UI        URL    https://127.0.0.1:<port>
	    1       UI        Scheme https
	    1       UI        User   ��
	    1       UI        Host   127.0.0.1
	    1       UI        Port   <port>
	    1       UI        Path   ��
	    
	but found (query options: "colnames") :
	    node_id  component  field   value                                                                                                                                                                                       
	    1        DB         URL     postgresql://root@127.0.0.1:<port>?application_name=cockroach&sslcert=test_certs%2Fclient.root.crt&sslkey=test_certs%2Fclient.root.key&sslmode=verify-full&sslrootcert=test_certs%2Fca.crt  
	    1        DB         Scheme  postgresql                                                                                                                                                                                  
	    1        DB         User    root                                                                                                                                                                                        
	    1        DB         Host    127.0.0.1                                                                                                                                                                                   
	    1        DB         Port    <port>                                                                                                                                                                                      
	    1        DB         Path    ��                                                                                                                                                                                           
	    1        UI         URL     https://127.0.0.1:<port>                                                                                                                                                                    
	    1        UI         Scheme  https                                                                                                                                                                                       
	    1        UI         User    ��                                                                                                                                                                                           
	    1        UI         Host    127.0.0.1                                                                                                                                                                                   
	    1        UI         Port    <port>                                                                                                                                                                                      
	    1        UI         Path    ��                                                                                                                                                                                           
	
logic_test.go:1804: testdata/logic_test/crdb_internal:191: too many errors encountered, skipping the rest of the input
--- FAIL: testrace/TestLogic (590.550s)
<autogenerated>:10: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic273841914
<autogenerated>:9: use -show-logs to present logs inline
--- FAIL: testrace/TestLogic/default (1.290s)
null
--- FAIL: testrace/TestLogic/default/crdb_internal (5.910s)
logic_test.go:1771: 
	testdata/logic_test/crdb_internal:174: 
	expected:
	    node_id component field  value
	    1       DB        URL    postgresql://root@127.0.0.1:<port>?sslcert=test_certs%2Fclient.root.crt&sslkey=test_certs%2Fclient.root.key&sslmode=verify-full&sslrootcert=test_certs%2Fca.crt
	    1       DB        Scheme postgresql
	    1       DB        User   root
	    1       DB        Host   127.0.0.1
	    1       DB        Port   <port>
	    1       DB        Path   ��
	    1       UI        URL    https://127.0.0.1:<port>
	    1       UI        Scheme https
	    1       UI        User   ��
	    1       UI        Host   127.0.0.1
	    1       UI        Port   <port>
	    1       UI        Path   ��
	    
	but found (query options: "colnames") :
	    node_id  component  field   value                                                                                                                                                                                       
	    1        DB         URL     postgresql://root@127.0.0.1:<port>?application_name=cockroach&sslcert=test_certs%2Fclient.root.crt&sslkey=test_certs%2Fclient.root.key&sslmode=verify-full&sslrootcert=test_certs%2Fca.crt  
	    1        DB         Scheme  postgresql                                                                                                                                                                                  
	    1        DB         User    root                                                                                                                                                                                        
	    1        DB         Host    127.0.0.1                                                                                                                                                                                   
	    1        DB         Port    <port>                                                                                                                                                                                      
	    1        DB         Path    ��                                                                                                                                                                                           
	    1        UI         URL     https://127.0.0.1:<port>                                                                                                                                                                    
	    1        UI         Scheme  https                                                                                                                                                                                       
	    1        UI         User    ��                                                                                                                                                                                           
	    1        UI         Host    127.0.0.1                                                                                                                                                                                   
	    1        UI         Port    <port>                                                                                                                                                                                      
	    1        UI         Path    ��                                                                                                                                                                                           
	
logic_test.go:1804: testdata/logic_test/crdb_internal:191: too many errors encountered, skipping the rest of the input
--- FAIL: testrace/TestLogic/distsql (2.830s)
null
--- FAIL: testrace/TestLogic/distsql/crdb_internal (7.470s)
logic_test.go:1771: 
	testdata/logic_test/crdb_internal:174: 
	expected:
	    node_id component field  value
	    1       DB        URL    postgresql://root@127.0.0.1:<port>?sslcert=test_certs%2Fclient.root.crt&sslkey=test_certs%2Fclient.root.key&sslmode=verify-full&sslrootcert=test_certs%2Fca.crt
	    1       DB        Scheme postgresql
	    1       DB        User   root
	    1       DB        Host   127.0.0.1
	    1       DB        Port   <port>
	    1       DB        Path   ��
	    1       UI        URL    https://127.0.0.1:<port>
	    1       UI        Scheme https
	    1       UI        User   ��
	    1       UI        Host   127.0.0.1
	    1       UI        Port   <port>
	    1       UI        Path   ��
	    
	but found (query options: "colnames") :
	    node_id  component  field   value                                                                                                                                                                                       
	    1        DB         URL     postgresql://root@127.0.0.1:<port>?application_name=cockroach&sslcert=test_certs%2Fclient.root.crt&sslkey=test_certs%2Fclient.root.key&sslmode=verify-full&sslrootcert=test_certs%2Fca.crt  
	    1        DB         Scheme  postgresql                                                                                                                                                                                  
	    1        DB         User    root                                                                                                                                                                                        
	    1        DB         Host    127.0.0.1                                                                                                                                                                                   
	    1        DB         Port    <port>                                                                                                                                                                                      
	    1        DB         Path    ��                                                                                                                                                                                           
	    1        UI         URL     https://127.0.0.1:<port>                                                                                                                                                                    
	    1        UI         Scheme  https                                                                                                                                                                                       
	    1        UI         User    ��                                                                                                                                                                                           
	    1        UI         Host    127.0.0.1                                                                                                                                                                                   
	    1        UI         Port    <port>                                                                                                                                                                                      
	    1        UI         Path    ��                                                                                                                                                                                           
	
logic_test.go:1804: testdata/logic_test/crdb_internal:191: too many errors encountered, skipping the rest of the input

```
Please assign, take a look and update the issue accordingly.

--- a/pkg/sql/logictest/testdata/logic_test/crdb_internal
+++ b/pkg/sql/logictest/testdata/logic_test/crdb_internal
@@ -174,19 +174,19 @@ select crdb_internal.force_retry(interval '0s')
 query ITTT colnames
 select node_id, component, field, regexp_replace(regexp_replace(value, '^\d+$', '<port>'), e':\\d+', ':<port>') as value from crdb_internal.node_runtime_info
 ----
-node_id component field  value
-1       DB        URL    postgresql://root@127.0.0.1:<port>?sslcert=test_certs%2Fclient.root.crt&sslkey=test_certs%2Fclient.root.key&sslmode=verify-full&sslrootcert=test_certs%2Fca.crt
-1       DB        Scheme postgresql
-1       DB        User   root
-1       DB        Host   127.0.0.1
-1       DB        Port   <port>
-1       DB        Path   ·
-1       UI        URL    https://127.0.0.1:<port>
-1       UI        Scheme https
-1       UI        User   ·
-1       UI        Host   127.0.0.1
-1       UI        Port   <port>
-1       UI        Path   ·
+node_id  component  field   value
+1        DB         URL     postgresql://root@127.0.0.1:<port>?application_name=cockroach&sslcert=test_certs%2Fclient.root.crt&sslkey=test_certs%2Fclient.root.key&sslmode=verify-full&sslrootcert=test_certs%2Fca.crt
+1        DB         Scheme  postgresql
+1        DB         User    root
+1        DB         Host    127.0.0.1
+1        DB         Port    <port>
+1        DB         Path    ·
+1        UI         URL     https://127.0.0.1:<port>
+1        UI         Scheme  https
+1        UI         User    ·
+1        UI         Host    127.0.0.1
+1        UI         Port    <port>
+1        UI         Path    ·
 
 user testuser
 

